apollo/tests/column.spec.js

```diff
@@ -5,40 +5,92 @@ const schema = fs.readFileSync('./schema/generated/prisma.graphql', 'utf8');
 
 const MyServer = mockServer(schema);
 
+const createStatusMutation = `
+  mutation CreateStatusMutation($name: String!, $projects: [String], $id: ID!, $labels: [String]) {
+    createStatus(name: $name, projects: $projects, id: $id, labels: $labels){
+      id
+      name
+      projects{
+        id
+        name
+      }
+      labels{
+        id
+        name
+        color
+      }
+    }
+  }
+`;
+
+const statusQuery = `
+  query StatusQuery {
+    statuses{
+      id
+      name
+      labels{
+        id
+        name
+        color
+      }
+      projects{
+        id
+        name
+      }
+      program{
+        id
+        name
+      }
+    }
+  }
+`;
+
+const deleteStatusMutation = `
+  mutation DeleteStatusMutation($id: id){
+    deleteStatus(id: $id){
+      id
+      name
+    }
+  }
+`;
+
+const updateStatusMutation = `
+  mutation UpdateStatusMutation($id: id, $name: String!){
+    updateStatus(id: $id, name: $name){
+      id
+      name
+    }
+  }
+`
+
 // Queries if there is a label
 describe('Column', () => {
   it('Column query is there', async () => {
     const server = MyServer;
     const query = `
-          {
-            columns {
-              name
-            }
-          }
-        `;
+    query {
+      statuses {
+        name
+      }
+    }`;
     await expect(server.query(query)).resolves.toBeTruthy();
     const { errors } = await server.query(query);
     expect(errors).not.toBeTruthy();
   });
 });
 
-describe('Column', () => {
-  it('Column can be updated', async () => {
+describe('Create Status', () => {
+  it('Status can be created', async () => {
     const server = MyServer;
 
-    const mutation = `
-    mutation {
-      createColumn(
-        data: 
-        {name:"Status 1",  program: { create: {name: "Labs 22"}}}){
-        name
-        id
-      }
-    }
-    `;
+    const createResponse = await context.prisma.createStatus({name: "Test Status"});
 
-    await expect(server.query(mutation)).toBeTruthy();
-    const { errors } = await server.query(mutation);
-    expect(errors).not.toBeTruthy();
+    expect(createResponse).toEqual({data: { name: "Test Status"}});
+  });
+});
+
+describe('Delete Status', () => {
+  it('deletes a status', async () => {
+    
   });
 });
```

Comment:
Pulled down with FE
Created labels, they persist! 
I get this error when running the tests

```
 FAIL  tests/column.spec.js (7.311s)
  ● Create Status › Status can be created

    ReferenceError: context is not defined

      84 |     const server = MyServer;
      85 | 
    > 86 |     const createResponse = await context.prisma.createStatus({name: "Test Status"});
         |                            ^
      87 | 
      88 |     expect(createResponse).toEqual({data: { name: "Test Status"}});
      89 |   });

      at Object.<anonymous> (tests/column.spec.js:86:28)
```